File: cliente/src/main/java/com/ClienteListener.java
package com;

import java.util.List;

public interface ClienteListener {
    void onMensajeRecibido(String emisor, String contenido);
    void onMensajePrivadoRecibido(String emisor, String contenido);
    void onListaUsuariosActualizada(List<String> usuarios);
    void onErrorRecibido(String mensajeError);
    void onDesconexion();
}

File: pytom/cliente_listener.py
class ClienteListener:
    def on_mensaje_recibido(self, emisor, contenido):
        pass

    def on_mensaje_privado_recibido(self, emisor, contenido):
        pass

    def on_lista_usuarios_actualizada(self, usuarios):
        pass

    def on_error_recibido(self, mensaje_error):
        pass

    def on_desconexion(self):
        pass
